src/client/components/LogView.tsx

```diff
@@ -82,7 +96,7 @@ class LogView extends Component<LogViewProps, LogViewState> {
           onClick={() => this.setState({ collapsed: !this.state.collapsed })}
         />
         <ReactJson
-          src={this.state.log}
+          src={{ shiftedTimezoneDate: shiftedTimezoneDate, ...this.state.log }}
```

Comment:
When both the `key` and the `value` have the same value, you don't need to repeat it:
This is how it should look like:
```src={{ shiftedTimezoneDate, ...this.state.log }}```

Reply:
Updated per comment.